tomcat-conf/Catalina/localhost/prweb.xml

```diff
@@ -73,6 +73,10 @@
   <Environment name="prconfig/dsm/services/stream/pyBaseLogPath" value="/opt/pega/kafkadata/data" type="java.lang.String" />
   <Environment name="prconfig/cloud/isNodeAgnosticAdminStudio" value="true" type="java.lang.String"/>
 
+  {{ if .Env.HZ_VERSION }}
+  <Environment name="prconfig/cluster/hazelcast/{{ .Env.HZ_VERSION }}/enabled" value="true" type="java.lang.String"/>
+  {{ end }}
```

Comment:
Is `prconfig/cluster/clientserver/server/hostname` not required to set? AFAIK this value is different for v4 to v5?

Reply:
Don't we need the below condition at this point? This flag should only be passed when we are running Hz in c/s mode.
`{{ if isTrue .Env.HZ_CLIENT_MODE }}`

Reply:
As discussed over the call All C/s related configs are already in prweb.

Reply:
After discussing with @vnihal72 , I have learned that the changes to this parameter will be made in the pull request for the Helm charts